tests/cli-colorspace.tap

```diff
@@ -14,8 +14,8 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-. ./common.shi
```

Comment:
Does these changes have any impact on the old build pipeline? I'm just thinking in case they ever want to accept this upstream.

Reply:
Maybe we can postpone making the tests work if this is a blocker? Feels like you're so close to getting this and the conan package done :)